src/vanilla/Model/MethodGroupTS.cs

```diff
@@ -29,6 +29,14 @@ protected MethodGroupTS(string name): base(name)
 
         public string MappersModuleName => TypeName.ToCamelCase() + "Mappers";
 
+        public string Imports
```

Comment:
I need to make use of this. The plan is to throw the logic in here to decide which import statements to emit--I think there's an existing method around MethodGroup or CodeModel that I should just edit or copy, though.